### ISSUE TITLE
Improve result evaluation of decsion tree model

### ISSUE THREAD

**Issue by jimmyhu4** [TravorLZH/pm25#2]
In `pm25_forecast.ipynb`, the prediction results on training set are plotted. It could be nice to add scatter plots to show the results on test set as well. In addition, it will be helpful to plot out the predicted time series and the true PM2.5 time series.
